src/components/intl/en-us.ts

```diff
@@ -0,0 +1,11 @@
+export default {
+  back: "Back",
+  close: "Close",
+  open: "Open",
+  expand: "Expand",
+  collapse: "Collapse",
+  next: "Next",
+  previous: "Previous",
+  tipGroupTitle: "Tips",
```

Comment:
I'm wondering if there would be any benefits, build-wise, in having individual bundles vs a single one.

Reply:
I think the benefit is not having the text "Close" defined a million times throughout component resources. But I see the issue if a user just wants to change one of them. Yes, we should research this more.